docs/getting-started/software-management/contributing-with-git.md

```diff
@@ -0,0 +1,164 @@
+
+If you don't have git yet, install as appropriate for your system.
+
+```sh
+# Run to see if/what version of git is installed
+git version
+```
+
+
+??? tip "Installing git for Windows, Mac, and Linux" 
+
+    | System  | Installation                                                                                                                                                                                               |
+    |---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | Windows | Download and install git for Windows from the [git-scm website (external)](https://git-scm.com/download/win).  If you have winget you may install git via `winget install --id Git.Git -e --source winget` |
+    | Mac     | Mac installs git automatically when you run it from the terminal.  If you have Homebrew, you may install git via `brew install git`.                                                                       |
+    | Linux   | Install git using your package manager. For example, `sudo apt install git` on Ubuntu.                                                                                                                     |
+
+
+```sh
+# Once git is installed, set your name and email:
+git config --global user.name "Your Name"
+git config --global user.email "emailaddress@example.com"
+```
+
+## Issues
+
+![Issue board on GitHub](../../assets/software-management/git-issue-board.png){ align=right width=450 }
+
+Your contribution should be related to an issue.  You can browse a list of issues on the repository page for each project. 
+
+A git issue is not necessarily something wrong, it may also be a feature to be added.
+
+!!! tip
+
+    If you are working on an issue you have found yourself, and the issue is not listed yet, please [report the issue](../../how-to-guides/software-management/guidelines-for-reporting-issues.md) and assign it to yourself.
+
+-----
+
+## Fork
+
+![Forking a repository on GitHub](../../assets/software-management/git-fork.png){ align=right width=450 }
+
+A ***fork*** is a copy of a repository.  The repo that has been forked from is usually called ***"upstream"***, and your fork/working copy is called ***"origin"*** by default.
+
+Your fork is your own working copy, where you can test your code or add new features.  While you are working on your contribution, you don't have to worry about unfinished code affecting the upstream repo; you are on your own fork.  Later, when your code is done, there is a review process before your code can be merged into the upstream repo.
+
+#### To make a fork:
+
+1. Click the fork button on the repository page
+1. Enter a name for your fork.
+    - If you want to contribute back to the original repo, you can just leave the name the same (your fork is differentiated by your username).
```

Comment:
This documentation has to be kept up to date, which has some associated cost. This documentation about the workflow is also duplicative of a number of other sources (e.g., https://gitprotect.io/blog/git-forking-workflow/, https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow).

It is more effective to maintain our own version of these or to curate and maintain links (that will be dead some day) to other documentation?